tt_metal/tools/profiler/op_profiler.hpp

```diff
@@ -385,7 +385,13 @@ inline std::string op_meta_data_serialized_json(
 
     j["optional_input_tensors"] = std::vector<json>{};
 
-    auto perfModel = operation_t::create_op_performance_model(operation_attributes, tensor_args, tensor_return_value);
```

Comment:
Do we want to write some more info on "how to perf model?"

Reply:
@xanderchin